docs/data/material/experimental-api/pigment-css/pigment-css.md

```diff
@@ -0,0 +1,338 @@
+  fontSize: '2rem',
+  color: '#9FADBC',
+  fontWeight: 'bold',
+  margin: '1rem',
+  variants: [
+    {
+      props: { variant: 'success' },
+      style: { color: '#23AD79' },
+    },
+    {
+      props: { size: 'small' },
+      style: { fontSize: '1.5rem' },
+    },
+  ],
+});
+```
+
+**Styling based on runtime values**
+
+When the value of a prop is unknown ahead of time, you can style your components based on runtime values:
+
+```js
+const Heading = styled('h1')({
+  color: ({ isError }) => (isError ? 'red' : 'black'),
+});
+```
+
+### Themes
+
+Pigment CSS supports theming to apply consistent styles and values across your application.
+You can create themes by defining them in your config file:
+
+```js
+import { withPigment } from '@pigment-css/nextjs-plugin';
+
+export default withPigment(nextConfig, {
+  theme: {
+    colors: {
+      primary: 'tomato',
+      secondary: 'cyan',
+    },
+    spacing: {
+      unit: 8,
+    },
+    typography: {
+      fontFamily: 'Inter, sans-serif',
+    },
+    // ...more keys and values, it's free style!
+  },
+});
+```
+
+To access your themes, use a callback with the `styled()` and `css()` APIs:
+
+```js
+const Heading = styled('h1')(({ theme }) => ({
+  color: theme.colors.primary,
+  fontSize: theme.spacing.unit * 4,
+  fontFamily: theme.typography.fontFamily,
+}));
+```
+
+#### CSS variables support
+
+Pigment CSS generates CSS variables from the theme values when they're wrapped by the `extendTheme` utility, creating a `vars` object:
+
+```js
+import { withPigment, extendTheme } from '@pigment-css/nextjs-plugin';
+
+export default withPigment(nextConfig, {
+  theme: extendTheme({
+    colors: {
+      primary: 'tomato',
+      secondary: 'cyan',
+    },
+    spacing: {
+      unit: 8,
+    },
+    typography: {
+      fontFamily: 'Inter, sans-serif',
+    },
+  }),
+});
+```
+
+#### Color schemes
+
+You can use the `colorSchemes` key within the `extendTheme` utility to assign different values based on different conditions, such as switching between dark mode and light mode:
+
+```js
+extendTheme({
+  colorSchemes: {
+    light: {
+      colors: {
+        background: '#f9f9f9',
+        foreground: '#121212',
+      },
+    },
+    dark: {
+      colors: {
+        background: '#212121',
+        foreground: '#fff',
+      },
+    },
+  },
+});
+```
+
+Pigment CSS uses the `prefers-color-scheme` media query by default to switch between color schemes:
+
+```js
+const colorScheme = css`
+  background-color: ${({ theme }) => theme.colorSchemes.dark.colors.background};
+
+  color: ${({ theme }) => theme.colorSchemes.dark.colors.foreground};
+
+  @media (prefers-color-scheme: light) {
+    background-color: ${({ theme }) => theme.colorSchemes.light.colors.background};
+    color: ${({ theme }) => theme.colorSchemes.light.colors.foreground};
+  }
+`;
+```
+
+You can also customize the behavior by providing a `getSelector` function:
+
+```js
+  extendTheme({
+    colorSchemes: {
```

Comment:
Wrong pragma and wrong indentation, should be:

````suggestion
```diff
 extendTheme({
   colorSchemes: {
````